packages/synapse-react-client/src/lib/style/components/_cards.scss

```diff
@@ -213,7 +213,6 @@ $text-color-lighter: #898989;
       display: inline-block;
       height: 40px;
       margin-right: 10px;
-      border-radius: 20px;
```

Comment:
Style "Download Dataset" button in NF Portal: 
main | feature
:---:|:---:
<img width="1259" alt="main_NF_downloaddataset" src="https://user-images.githubusercontent.com/26949006/231906277-bb3b2332-9019-4c08-9c6c-b5294f37d263.png"> | <img width="1253" alt="feature_NF_downloaddataset" src="https://user-images.githubusercontent.com/26949006/231906292-ac0ae2f5-2fda-4cd1-852c-18b1a5e3fa76.png">